sdcm/tester.py

```diff
@@ -1690,110 +1691,48 @@ def get_cluster_k8s_gke(self):  # pylint: disable=too-many-branches
-            num_nodes=self.params.get("n_db_nodes"),
-            instance_type=self.params.get('instance_type_db'),
-            role_arn=self.params.get('eks_nodegroup_role_arn'),
-            disk_size=self.params.get('root_disk_size_db'),
-            k8s_cluster=k8s_cluster
-        )
-        k8s_cluster.deploy_node_pool(scylla_pool, wait_till_ready=False)
-
-        loader_pool = None
-        if self.params.get("n_loaders"):
-            loader_pool = eks.EksNodePool(
-                name=k8s_cluster.LOADER_POOL_NAME,
-                num_nodes=self.params.get("n_loaders"),
-                instance_type=self.params.get("instance_type_loader"),
-                role_arn=self.params.get('eks_nodegroup_role_arn'),
-                disk_size=self.params.get('root_disk_size_monitor'),
-                k8s_cluster=k8s_cluster)
-            k8s_cluster.deploy_node_pool(loader_pool, wait_till_ready=False)
-
-        monitor_pool = None
-        if self.params.get('k8s_deploy_monitoring'):
-            monitor_pool = eks.EksNodePool(
-                name=k8s_cluster.MONITORING_POOL_NAME,
-                num_nodes=self.params.get("k8s_n_monitor_nodes") or self.params.get("n_monitor_nodes"),
-                instance_type=self.params.get(
-                    "k8s_instance_type_monitor") or self.params.get("instance_type_monitor"),
-                role_arn=self.params.get('eks_nodegroup_role_arn'),
-                disk_size=self.params.get('root_disk_size_monitor'),
-                k8s_cluster=k8s_cluster
-            )
-            k8s_cluster.deploy_node_pool(monitor_pool, wait_till_ready=False)
-        k8s_cluster.wait_all_node_pools_to_be_ready()
-        k8s_cluster.configure_ebs_csi_driver()
-
-        k8s_cluster.deploy_cert_manager(pool_name=k8s_cluster.AUXILIARY_POOL_NAME)
-        if self.params.get('k8s_enable_tls'):
-            k8s_cluster.deploy_ingress_controller(pool_name=k8s_cluster.AUXILIARY_POOL_NAME)
-        k8s_cluster.deploy_scylla_operator()
-        k8s_cluster.prepare_k8s_scylla_nodes(node_pools=scylla_pool)
-        if self.params.get('use_mgmt'):
-            k8s_cluster.deploy_scylla_manager(pool_name=k8s_cluster.AUXILIARY_POOL_NAME)
-
-        for i in range(k8s_cluster.tenants_number):
+        for i in range(self.k8s_clusters[0].tenants_number):
             self.db_clusters_multitenant.append(eks.EksScyllaPodCluster(
-                k8s_clusters=[k8s_cluster],
+                k8s_clusters=self.k8s_clusters,
                 scylla_cluster_name=self.params.get("k8s_scylla_cluster_name") + (f"-{i + 1}" if i else ""),
                 user_prefix=(f"{i + 1}-" if i else "") + self.params.get("user_prefix"),
                 n_nodes=self.params.get("k8s_n_scylla_pods_per_cluster") or self.params.get("n_db_nodes"),
                 params=deepcopy(self.params),
-                node_pool_name=scylla_pool.name,
+                node_pool_name=self.k8s_clusters[0].SCYLLA_POOL_NAME,
```

Comment:
we assume pool names would be identical ?
or this PR doesn't yet have the scylla pods spread in both k8s cluster ?

Reply:
Only `auxiliary/default` node pool name differs for the same thing.
The `scylla-pool` name is defined only by us and there is no any need to make it be different for some backends.